### PR TITLE
Add disk space issue to common issues.

### PR DESCRIPTION
**Pull request by froboy**
This adds some additional bug fix steps [from Slack](https://drupal.slack.com/archives/C6GPEEEV8/p1613526619020000) regarding out of disk issues.

### CODE REVIEW

File: docs/content/troubleshooting/common-issues.md [hunk 1]
```diff
@@ -477,4 +477,22 @@ Run your composer command beginning with...
...
+
+## Issue 19. Out of disk space
+
+Out of disk issues may manifest in many ways. One is:
+
+```
+ERROR 1114 (HY000) at line xxxx: The table 'xxxx' is full
+```
+
+You can diagnose this issue with `fin exec df -lh`. If a disk shows close to 100% usage it's time to expand it.
```
Comment (shelane) [inline docs/content/troubleshooting/common-issues.md:490]: Add a comma => If a disk shows close to 100% usage, it's time to expand it.

**Review by shelane — CHANGES_REQUESTED** [review]
Just one minor edit requested.

**Comment by lmakarov** [comment]
If the VM disk gets full and it feels like you don't have any projects that should legitimately occupy all that space, then instead of increasing the disk size you could also do some housekeeping with `fin cleanup`

```
❯ fin help cleanup

Remove all unused Docker images, and Docksal related volumes and orphaned containers to save disk space.
Orphaned are those containers which project folders were deleted from the filesystem,
but the containers still linger in the Docker.

Usage: cleanup [options]

Options:
  --images                 	Docker images cleanup Wizard
  --hard                   	Remove ALL stopped containers even unrelated to Docksal (potentially destructive operation)

Examples:
  fin cleanup              	Regular cleanup
  fin cleanup --images     	Docker Images cleanup wizard, helping to navigate and delete unused ones
  fin cleanup --hard       	Run hard cleanup removing all stopped Docker containers
```

Another (extremely efficient option) is to reset the VM altogether 🤠

File: docs/content/troubleshooting/common-issues.md (hunk 1 above)
Comment (froboy) [reply docs/content/troubleshooting/common-issues.md:490]:
```suggestion
You can diagnose this issue with `fin exec df -lh`. If a disk shows close to 100% usage, it's time to expand it.
```

Comment (froboy) [inline docs/content/troubleshooting/common-issues.md:490]:
```suggestion
You can diagnose this issue with `fin exec df -lh`. If a disk shows close to 100% usage, you'll need to either [clean up your projects](https://docs.docksal.io/core/vm/#free-up-space-in-virtualbox-vm-docker-for-mac-or-docker-for-windows) or expand the disk.
```

Comment (froboy) [reply docs/content/troubleshooting/common-issues.md]: @shelane thanks! I've addressed this below.

**Comment by froboy** [comment]
@lmakarov thanks, I added a link to the more detailed cleanup docs. In my case, I'm doing support work and juggling a lot of projects at once, but `cleanup` is definitely a good first step for a lot of folks.

**Review by shelane — APPROVED** [review]

**Pull request merged**